### DISCUSSION TITLE
Sonos S1 and S2 at same time?

### DISCUSSION THREAD

**Discussion by maxandersen** [music-assistant/support#2246] [Q&A] [answered]
I have 10 speakers on S1 and 3 on S2. 

I would love being able to use both at the same time but when I add sonos as service it seem to only pick S1. 

Any plans on supporting both?

**Comment by OzGav** [comment, marked as answer]
Yes that will be looked at next week. I understand if you have all S1 or all S2 then it works ok but when they are mixed in the network problems occur.
